docs/versioning.rst

```diff
@@ -0,0 +1,156 @@
+**********
+Versioning
+**********
+
+
+Current Release Stages
+======================
+
+.. list-table:: Releases
+   :widths: 20 30 30 20
+   :header-rows: 1
+
+   * - Release
+     - Runtimes
+     - Minimum Trace Agent
+     - Stage
+   * - 0.x
+     - CPython: 2.7, 3.5–3.10
+     - 
+     - MAINTENANCE [2022-03-01]
+   * - 1.x
+     - CPython: 2.7, 3.5+
+     - 7.28
+     - CURRENT
+
+
+Runtimes
+========
+
+
+.. list-table:: Supported runtime versions
+   :widths: 60 40
+   :header-rows: 1
+
+   * - Runtime
+     - Runtime version
```

Comment:
should we include "tested in CI" same as OS?

if we ever decide to stop testing 3.5 but want to keep supporting it... would we ever do that? or maybe just good to be explicit?

Reply:
I am not totally happy with these tables as is.

We really want a table of `OS x Arch x Runtime` with whether the combination is tested in CI and we distribute wheels. wdyt?

Reply:
yeah, I agree, that sounds great.

verbose, but we can figure out a concise way to represent it... 🤞🏻